src/lock-size.js

```diff
@@ -0,0 +1,15 @@
+import once from 'once';
+
+const lockSize = (element) => {
+    const boundingRect = getComputedStyle(element);
+
+    element.style.height = boundingRect.height;
+    element.style.width = boundingRect.width;
+
+    return once(() => {
```

Comment:
This sounds like a big hammer, why did you have to do this? 😄

Reply:
I will remove it since it's not needed.